tests/test_full_text_search.py

```diff
@@ -192,6 +202,12 @@ def test_search_by_name_only(self):
         assert set([r.record_id for r in results]) == \
```

Comment:
Do you have equivalent unit test using romanization with unidecode? Please add one if missing.